server/models/rating.js

```diff
@@ -11,7 +11,7 @@ module.exports = (sequelize, DataTypes) => {
       primaryKey: true,
     },
     rating_value: {
-      type: DataTypes.integer,
+      type: DataTypes.INTEGER,
```

Comment:
I noticed you sneaked this in 👀

Reply:
When I pushed I noticed this typo was causing an error in travis

Reply:
No worries 🙂